stackslib/src/core/mempool.rs

```diff
@@ -803,6 +803,16 @@ const MEMPOOL_SCHEMA_6_NONCES: &'static [&'static str] = &[
     "#,
 ];
 
+const MEMPOOL_SCHEMA_7_TIME_ESTIMATES: &'static [&'static str] = &[
+    r#"
+    -- ALLOW NULL
+    ALTER TABLE mempool ADD COLUMN time_estimate_ms NUMBER;
```

Comment:
Wouldn't the column affinity be `INTEGER`?

Reply:
Fixed in [6f60813](https://github.com/stacks-network/stacks-core/pull/5238/commits/6f60813c91468dce9971a4a859c98f86dc31a1e0)